go/vt/vtgate/planbuilder/operators/route_planning.go

```diff
@@ -436,11 +435,11 @@ func mergeOrJoin(ctx *plancontext.PlanningContext, lhs, rhs ops.Operator, joinPr
 
 	if len(joinPredicates) > 0 && requiresSwitchingSides(ctx, rhs) {
 		if !inner {
-			return nil, vterrors.Errorf(vtrpcpb.Code_UNIMPLEMENTED, "unsupported: LEFT JOIN not supported for derived tables")
+			return nil, vterrors.VT12001("LEFT JOIN not supported for derived tables")
```

Comment:
```suggestion
			return nil, vterrors.VT12001("LEFT JOIN with derived tables")
```

Reply:
vterrors.VT12001("subqueries in GROUP BY")

Reply:
resolved in [f2351bb](https://github.com/vitessio/vitess/pull/10738/commits/f2351bba65f347a8e5ce72ff87757ccaf20115af)